h/browser/chrome/test/hypothesis-chrome-extension-test.js

```diff
@@ -67,11 +69,16 @@ describe('HypothesisChromeExtension', function () {
       setState: sandbox.spy(),
       activate: sandbox.spy(),
       deactivate: sandbox.spy(),
+      updateBadge: sandbox.spy(),
     };
     fakeSidebarInjector = {
       injectIntoTab: sandbox.stub().returns(Promise.resolve()),
       removeFromTab: sandbox.stub().returns(Promise.resolve()),
     };
+    fakeUriInfo = sandbox.stub().returns(
+        Promise.resolve({total: 3, blocked: false}));
+    // Fix a proxyquire crash due to a PhantomJS bug.
```

Comment:
If you need a workaround for a bug, we should always reference the upstream issue in the bug tracker or explicitly describe the issue. In this case, you can reference https://github.com/thlorenz/proxyquireify/issues/44